src/main/java/core/basesyntax/service/WriteService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface WriteService {
```

Comment:
fix name according to conventions
https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names